client/src/app/shared/models/poll/view-poll.ts

```diff
@@ -0,0 +1,141 @@
+    protected _collection = Poll.COLLECTION;
+
+    public get pollClassType(): PollClassType {
+        return this.content_object?.collection as PollClassType;
+    }
+
+    public get isAssignmentPoll(): boolean {
+        return this.content_object?.collection === PollClassType.Assignment;
+    }
+
+    public get isMotionPoll(): boolean {
+        return this.content_object?.collection === PollClassType.Motion;
+    }
+
+    public get pollmethodVerbose(): string {
+        return PollMethodVerbose[this.pollmethod];
+    }
+
+    public get percentBaseVerbose(): string {
+        return PollPercentBaseVerbose[this.onehundred_percent_base];
+    }
+
+    public get pollClassTypeVerbose(): string {
+        return PollClassTypeVerbose[this.pollClassType];
+    }
+
+    public get parentLink(): string {
+        const routeFragments = this.content_object_id.split('/');
+        return `/${routeFragments[0]}s/${routeFragments[1]}`;
+    }
+
+    public get stateVerbose(): string {
+        return PollStateVerbose[this.state];
+    }
+
+    public get nextStateActionVerbose(): string {
+        return PollStateChangeActionVerbose[this.nextState];
+    }
+
+    public get typeVerbose(): string {
+        return PollTypeVerbose[this.type];
+    }
+
+    public get majorityMethodVerbose(): string {
+        return MajorityMethodVerbose[this.majority_method];
+    }
+
+    public get hasVote(): boolean | null {
+        if (!this.user_has_voted && this.canBeVotedFor()) {
+            return false;
+        } else if (this.user_has_voted) {
+            return true;
+        } else {
+            return null;
+        }
+    }
+
+    public canBeVotedFor(): boolean {
+        console.warn('TODO: Not implemented!');
+        return true;
+    }
+
+    public anySpecialVotes(): boolean {
+        return this.result.yes < 0 || this.result.no < 0 || this.result.abstain < 0;
+    }
+
+    protected getDecimalFields(): string[] {
+        return Poll.DECIMAL_FIELDS;
+    }
+
+    public get result(): ViewOption {
+        if (this.global_option && this.type !== PollType.Analog) {
+            return this.global_option;
+        }
+        if (this.options) {
+            return this.options[0];
+        }
+        return null;
+    }
+
+    public get hasVotes(): boolean {
+        return !!this.result && !!this.result.votes.length && this.result.votes.some(vote => vote.weight > 0);
+    }
+
+    public hasVotedId(userId: number): boolean {
+        return this.user_has_voted_for_delegations?.includes(userId);
+    }
+
+    public getContentObject(): BaseViewModel {
+        return this.content_object;
+    }
+
+    public getSlide(): ProjectorElementBuildDeskriptor {
+        return {
+            getBasicProjectorElement: options => ({
+                name: Poll.COLLECTION,
+                id: this.id,
+                getNumbers: () => ['name', 'id']
+            }),
+            slideOptions: [],
+            projectionDefaultName: 'poll',
```

Comment:
Use the `ProjectionDefault` enum. There are special defualts for assignment and motion polls.